user/pages/03.commerce2/02.developer-guide/09.promotions/02.create-a-condition/docs.md

```diff
@@ -4,4 +4,222 @@ taxonomy:
     category: docs
 ---
 
+Commerce conditions are used in when creating a Promotion or a Payment Gateway 
+for your store. 
+
+In this context, conditions are a set of plugins unique to the 
+commerce project. They are not connected in function or code to conditions in 
+core such as: Drupal\Core\Database\Query\Condition. 
+
+A promotion condition evaluated only when a coupon is applied to an order.
+
+
+Minimum requirements to build a condition:
+---
+
+As you can see from comparing `ConditionInterface` with `ConditionBase`, 
+there is only one function required when extending ConditionBase to fulfill 
+the ConditionInterface prescription.
+
+```php
```

Comment:
There's a huge amount of code here. If it's helpful to show the code, be really specific about it. Show as little of the code as possible specific to what you want to point out.

Reply:
Having all this code would render the documentation obsolete the moment anything small changes.